### ISSUE TITLE
Editor reports an error with a valid security definition

### ISSUE THREAD

**Issue by greenstatic** [swagger-api/swagger-editor#1599]
| Q                                   | A
| ----------------------------------- | -------
| Bug or feature request?             | Bug 
| Which Swagger/OpenAPI version?      | 3.0.0
| Which Swagger-Editor version?       | The one currently online (editor.swagger.io)
| How did you install Swagger-Editor? | /
| Which broswer & version?            | Chrome (63.0) & Safari (11.0.2)
| Which operating system?             |  macOS High Sierra

### Demonstration API definition
```yaml
openapi: "3.0.0"
info:
  title: Pets Demo
  version: v1
paths:
  /pets:
    get:
      operationId: petsList
      summary: List of all pets
      security: 
        - JWT: []
      responses:
        '200':
          description: '200 response'
components:
  securitySchemes:
    JWT:
      type: http
      scheme: bearer
      bearerFormat: JWT
```

### Expected Behavior
The editor should not complain about an error, specifically:
"Semantic error at paths./pets.get.security.0
Security requirements must match a security definition"

### Current Behavior
Reports an error, however upon clicking on the little lock icon (next to the rest endpoint) the Available authorizations picks up the JWT security scheme.

This error started appearing today, a few days ago it was fine.

### Docs
The JWT bearer security format is documented [here](https://swagger.io/docs/specification/authentication/bearer-authentication/).

**Comment by nickzao** [comment]
+1. 
This problem appeared 2 days ago. Probably after this commit.
https://github.com/swagger-api/swagger-editor/commit/a901c7126dd0cb9252ec833c14cb6f4870803fe9#diff-6ff4c05e82f8bd0b8d107c8ba523a5ee

It seems to me that the check works correctly on the swagger 2.0 specifications and gives errors for the valid openapi 3.0 specifications.

**Comment by shockey** [comment]
@nickzao, yep, this is a bug. Looks like some of the new validators aren't correctly guarded to only process Swagger 2 definitions - oof!

**Comment by shockey** [comment]
Hi @greenstatic! I've released `swagger-editor@3.2.1` which includes a fix for this bug.

Thanks for the report!

**Comment by greenstatic** [comment]
@shockey thanks for the fast fix!
